keras_cv/models/segmentation/segment_anything/sam_mask_decoder.py

```diff
@@ -0,0 +1,230 @@
+# Copyright 2023 The KerasCV Authors
```

Comment:
I'd probably put this layer under sam layers

Reply:
Done.